robusta_krr/main.py

```diff
@@ -216,7 +222,16 @@ def run_strategy(
                     rich_help_panel="Logging Settings",
                 ),
                 show_cluster_name: bool = typer.Option(
-                    False, "--show-cluster-name", help="In table output, always show the cluster name even for a single cluster", rich_help_panel="Output Settings"
+                    False,
+                    "--show-cluster-name",
+                    help="In table output, always show the cluster name even for a single cluster",
+                    rich_help_panel="Output Settings",
+                ),
+                show_severity: bool = typer.Option(
+                    True,
+                    " /--exclude-severity",
```

Comment:
does the ` /` prefix means something, or it's a typo?

Reply:
It is since i want to set the False flag.
When using bool option the format is {true_value/false_value} and since by default is true I only wanted the false value.